packages/atomic/src/components/insight/user-actions/atomic-insight-user-actions-timeline/e2e/atomic-insight-user-actions-timeline.e2e.ts

```diff
@@ -0,0 +1,173 @@
+import {test, expect} from './fixture';
+
+const followingSessionActions = [
+  {
+    name: 'CUSTOM',
+    value:
+      '{"event_type":"example","event_value":"exampleCustomAction","origin_level_1":"default"}',
+    time: '1823680000000',
+  },
+];
+
+const caseCreationSessionActions = [
+  {
+    name: 'CUSTOM',
+    value:
+      '{"event_type":"errors","event_value":"play","origin_level_1":"default","origin_level_2":"default"}',
+    time: '1723680000001',
+  },
+  {
+    name: 'CUSTOM',
+    value:
+      '{"event_type":"errors","event_value":"play","origin_level_1":"default","origin_level_2":"default"}',
+    time: '1723680000002',
+  },
+  {
+    name: 'CUSTOM',
+    value:
+      '{"event_type":"errors","event_value":"play","origin_level_1":"default","origin_level_2":"default"}',
+    time: '1723679999999',
+  },
+];
+
+const precedingSessionActions = [
+  {
+    name: 'CUSTOM',
+    value:
+      '{"event_type":"example","event_value":"exampleCustomAction","origin_level_1":"default"}',
+    time: '1623680000000',
+  },
+];
+
+const exampleUserActions = [
+  ...followingSessionActions,
+  ...caseCreationSessionActions,
+  ...precedingSessionActions,
+];
+
+const exampleUserId = 'exampleUserId';
+const exampleTicketCreationDate = encodeURIComponent('2024-08-15');
+
+test.describe('default', () => {
+  test.describe('when user actions data is found', () => {
+    test.beforeEach(async ({userActionsTimeline, page}) => {
+      await userActionsTimeline.load({
+        args: {
+          userId: exampleUserId,
+          ticketCreationDate: exampleTicketCreationDate,
+        },
+      });
+      await userActionsTimeline.mockUserActions(page, exampleUserActions);
+    });
+
+    test.only('should display the case creation session', async ({
+      userActionsTimeline,
+      page,
+    }) => {
+      await expect(userActionsTimeline.activeSession).toBeVisible();
+      await page.waitForTimeout(35000);
+    });
+
+    test('should display the show following sessions button', async ({
+      userActionsTimeline,
+    }) => {
+      await expect(
+        userActionsTimeline.showFollowingSessionsbutton
```

Comment:
```suggestion
        userActionsTimeline.showFollowingSessionsButton
```

and for the other ones as well